_studio/shared/umc/codec/vc1_dec/src/umc_vc1_video_decoder.cpp

```diff
@@ -310,14 +310,14 @@ Status VC1VideoDecoder::ContextAllocation(uint32_t mbWidth,uint32_t mbHeight)
     {
 
         uint8_t* ptr = NULL;
-        ptr += mfx::align_value<uint32_t>(sizeof(VC1Context));
-        ptr += mfx::align_value<uint32_t>(sizeof(VC1VLCTables));
-        ptr += mfx::align_value<uint32_t>(sizeof(int16_t)*mbHeight*mbWidth*2*2);
-        ptr += mfx::align_value<uint32_t>(mbHeight*mbWidth);
-        ptr += mfx::align_value<uint32_t>(sizeof(VC1TSHeap));
+        ptr += mfx::align2_value<uint32_t>(sizeof(VC1Context));
+        ptr += mfx::align2_value<uint32_t>(sizeof(VC1VLCTables));
+        ptr += mfx::align2_value<uint32_t>(sizeof(int16_t)*mbHeight*mbWidth*2*2);
+        ptr += mfx::align2_value<uint32_t>(mbHeight*mbWidth);
+        ptr += mfx::align2_value<uint32_t>(sizeof(VC1TSHeap));
         if(m_stCodes == NULL)
         {
-            ptr += mfx::align_value<uint32_t>(START_CODE_NUMBER*2*sizeof(uint32_t)+sizeof(MediaDataEx::_MediaDataEx));
+            ptr += mfx::align2_value<uint32_t>(START_CODE_NUMBER*2*sizeof(uint32_t)+sizeof(MediaDataEx::_MediaDataEx));
```

Comment:
Do we still need explicit `uint32_t` here and above?

Reply:
no